ouroboros/chainsync/v5/types_test.go

```diff
@@ -28,15 +28,32 @@ func TestV5(t *testing.T) {
 		rawData, err := os.ReadFile("test_data/Tx_v6.json")
 		assert.Nil(t, err)
 
-		var expected chainsync.Tx
-		err = json.Unmarshal(rawData, &expected)
+		var expectedV6 chainsync.Tx
+		err = json.Unmarshal(rawData, &expectedV6)
 		assert.Nil(t, err)
-		actual := TxFromV6(expected)
+		v5Conversion := TxFromV6(expectedV6)
+		v6Conversion := v5Conversion.ConvertToV6()
 
-		// Signature: feb447eca6b819f88b4b6aac81a97200121dd4451bfccb65f73d31de296b5402eead42848808633ad1d4afbfad3a2f1967ce65516adca5cd373673f758a9c096
+		var bootstrapV5Sigs []chainsync.Signature
+		for _, x := range v5Conversion.Witness.Bootstrap {
+			var sig chainsync.Signature
+			json.Unmarshal(x, &sig)
+			bootstrapV5Sigs = append(bootstrapV5Sigs, sig)
+		}
+
+		network := "\"mainnet\""
```

Comment:
that's because it's a `json.RawMessage`, which is the raw bytes from JSON, which includes the quotes. When converting to v6, if v6 uses string, just `json.Unmarshal` into a string variable to remove the quotes.